src/pykoala/cubing.py

```diff
@@ -619,6 +622,30 @@ def rss_intensity(self):
     def rss_variance(self):
         return np.reshape(self.variance, (self.variance.shape[0], self.variance.shape[1]*self.variance.shape[2])).T
 
+    @classmethod
+    def from_fits(cls, path, hdul_extension_map=None, **kwargs):
```

Comment:
I wonder whether we should again follow the same structure as in `RSS`, i.e. instrument-specific cubes (even if we only implement `KOALA_Cube` for the time being).

Reply:
It's doable, but I see that feature as a mid-term addition once the corrections are well polished.